andela/mmdp_site
# MMDP-322 Implement search and refractor's research section

**What does this PR do?**

Implements search feature on the research section of the resources page

**Description of Task to be completed?**

- [x] An administrator should be able to search for a document displayed in the 'Research' section of the 'Resources' page by entering search criteria in the search box
- [x] Refractor research list implementation to use common components
- [x] Fix failing tests on users saga and component

**How should this be manually tested?**

- Checkout to this branch `ft-search-research-MMDP-332`
- RUN `yarn run:server` to start the backend server
- RUN `yarn client:dev` to start the frontend client
- Navigate to the research page, you should get a list of research, type in the title of the research you are looking for in the search box and click the search button. The research documents that match the search will appear in the list.

**Any background context you want to provide?**

 - none

**What are the relevant Jira issues?**

[MMDP-322](https://viisaus.atlassian.net/browse/MMDP-322)

**Screenshots (if appropriate)**

<img width="1440" alt="screen shot 2019-02-27 at 9 24 23 am" src="https://user-images.githubusercontent.com/9039613/53476100-e2a9b900-3a71-11e9-85c5-0d562817b956.png">
<img width="1440" alt="screen shot 2019-02-27 at 9 33 51 am" src="https://user-images.githubusercontent.com/9039613/53476839-a11a0d80-3a73-11e9-80ea-21d463374b98.png">


**Questions:**
